app/services/atmosphere/vmt_migrator.rb

```diff
@@ -0,0 +1,41 @@
+require 'migratio/worker/openstack_amazon_migrator'
+require 'migratio/worker/openstack_openstack_migrator'
+
+module Atmosphere
+  class VmtMigrator
+    def initialize(virtual_machine_template,
+                   source_compute_site, destination_compute_site)
+      @virtual_machine_template = virtual_machine_template
+      @source_compute_site = source_compute_site
+      @destination_compute_site = destination_compute_site
+    end
+
+    def select_migrator
+      case @destination_compute_site.technology
+      when 'aws'
+        migrator_class = Migratio::Worker::OpenstackAmazonMigrator
+      when 'openstack'
+        migrator_class = Migratio::Worker::OpenstackOpenstackMigrator
+      end
+      migrator_class
```

Comment:
this can be simplified into:

```ruby 
def select_migrator
  case @destination_compute_site.technology
  when 'aws'
    Migratio::Worker::OpenstackAmazonMigrator
  when 'openstack'
    Migratio::Worker::OpenstackOpenstackMigrator
  end
```